pkg/pillar/hypervisor/kvm.go

```diff
@@ -462,20 +462,135 @@ func (ctx kvmContext) Task(status *types.DomainStatus) types.Task {
 	}
 }
 
-// TBD: Have a better way to calculate this number.
-// For now it is based on some trial-and-error experiments.
-// Container limit is reduced to 100MiB.
-func minVMMOverhead(config types.DomainConfig) int64 {
+func estimatedVMMOverhead(domainName string, config types.DomainConfig, aa *types.AssignableAdapters) (int64, error) {
```

Comment:
What about adding a test to check that it works correctly?

Reply:
I thought about it, but I need somehow to feed /sys/pci/devices/<dev>/resources to test MMIO part. otherwise I can add a test.

Reply:
You can just mock that file and give a parameter to `estimatedVMMOverhead`